projecttacoma/bulk-submit-demo
# Kickoff Preview Panel

## Summary
Builds out an area for previewing the kickoff request for bulk data. Shows the URL, body and headers for running a bulk $submit-data operation. Provides a currently non-functional button for kicking off the request.

### New Behavior
Kickoff request can be previewed but not executed. On selecting a measure and adding an exportUrl the kickoff request will be populated.

### Code Changes
- KickoffRequestView set of components with KickoffRequestPanel being the top level component.
- Replaced react-json-view with Prism code highlighting view in Data Requirements view.
- Changed selectedMeasure recoil value to have both the measure `id` and `url`.

## Testing Guidance
- Start dev server.
- Note no info is shown in kickoff view.
- Select a measure, still no info.
- Type something into the `export url` box.
- Check that kickoff body and header tabs are populated with valid $submit-data request information.
